scripts/openbsd/init.sh

```diff
@@ -42,7 +45,7 @@ fi
 # XXX this patch should be applied in our ports tree, but we desparately need
 # package builder first. remove this when the builder is ready.
 #
-if [ `ansible --version | head -n 1 | cut -d' ' -f 2` == '2.3.2.0' ]; then
+if [ `ansible --version | head -n 1 | cut -d' ' -f 2` == '2.3.2.0' -a `uname -r` != '6.2' ]; then
```

Comment:
our ports tree has been patched for `6.2`?

Reply:
have not looked at the `patches` directory yet, but the package in `OPENBSD_6_2` has passed specs that check the bug in question.